index.js

```diff
@@ -70,6 +73,16 @@ const create = (win, options) => {
 					electron.shell.openExternal(url.toString());
 				}
 			}),
+			searchWithOther: decorateMenuItem({
+				id: 'searchWithOther',
+				label: `Search &with ${addSearchWithOtherHasValues ? options.addSearchWithOther.title : ''}`,
+				visible: hasText,
+				click() {
+					const url = new URL(addSearchWithOtherHasValues ? options.addSearchWithOther.url : '');
+					url.searchParams.set('q', props.selectionText);
```

Comment:
You are incorrectly assuming all search engines use `q`. Please follow what was discussed in the original issue.

Reply:
Sorry, I misread the doc, updating now